VIDEO_2_SCRIPT.md

```diff
@@ -0,0 +1,157 @@
+
```

Comment:
<img width="600" alt="Screen Shot 2022-02-03 at 8 18 01 PM" src="https://user-images.githubusercontent.com/19964402/152470987-7a267abe-60fc-4e2a-bc95-5f93d6d5989a.png">

Ok, just like in PR #8, I checked this script text with Grammarly and I only found a couple of spots that seemed to potentially flag plagiarism. Thought you'd want to know the result of running this script too 😀👍🏽👍🏽

Reply:
Yes, of course, thank you very much 🙂